examples/suit_update/tests/01-run.py

```diff
@@ -66,13 +66,6 @@ def publish(server_dir, server_url, app_ver, keys='default', latest_name=None):
     assert not subprocess.call(cmd)
 
 
-def wait_for_update(child):
-    return child.expect([r"riotboot_flashwrite: processing bytes (\d+)-(\d+)",
-                         "riotboot_flashwrite: riotboot flashing "
-                         "completed successfully"],
-                        timeout=UPDATING_TIMEOUT)
```

Comment:
This change is not justified properly. And is not robust, this makes the test fail when updating over the air since the udpate takes longer than 60s.

Reply:
> This change is not justified properly

What kind of justification do you expect ?

> And is not robust, this makes the test fail when updating over the air since the udpate takes longer than 60s.

The previous version can only work with the previous version: one message printed for each downloaded chunk. With the progress bar, this is another story. So now only the "Download success" message is checked.

Is the automatic test also supposed to work with over the air download (e.g. not only via ethos) ?

Reply:
> Is the automatic test also supposed to work with over the air download (e.g. not only via ethos) ?

It did before. At least at the time of the PR, that is why there was not an absolute timeout to handle the different times according to link speed and firmware size.